src/index.html

```diff
@@ -1,13 +1,403 @@
+                    alt="Headphones & earbuds"
+                    class="headphones__photo"
+                  >
+                  </div>
+                </div>
+
+                <div class="grid__item grid__item--2-2 grid__item--onIpad-3-6 grid__item--onDesktop-5-12">
+                  <div class="headphones__photo-container">
+                    <img
+                    src="./images/audio-second.jpg"
+                    alt="Headphones & earbuds"
+                    class="headphones__photo"
+                  >
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="headphones__description">
+              Audio sunglasses
+            </div>
+          </div>
+          <!-- button -->
+          <div class="grid">
+            <div class="grid__item grid__item--1-2 grid__item--onIpad-2-5 grid__item--onDesktop-5-8">
+              <a href="#" class="button">
+                View all
+              </a>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- why bose -->
+      <section class="page__section whyBose" id="whyBose">
+        <div class="container">
+          <div class="grid">
+            <div class="grid__item grid__item--1-2 grid__item--onIpad-1-3 grid__item--onDesktop-1-5">
+              <h2 class="whyBose__title">
+                Why buy direct
+                <br>
+                from Bose
+              </h2>
+            </div>
+
+            <div class="grid__item grid__item--1-2 grid__item--onIpad-4-6 grid__item--onDesktop-6-10">
+              <p class="whyBose__paragraph whyBose__paragraph--important">
+                Free 2-day shipping and returns
+              </p>
+
+              <p class="whyBose__paragraph whyBose__paragraph--important">
+                90-day risk-free trial
+              </p>
+
+              <p class="whyBose__paragraph whyBose__paragraph--important">
+                World class customer service
+              </p>
+
+              <p class="whyBose__paragraph whyBose__paragraph--important">
+                My Bose account management
+              </p>
+
+              <p class="whyBose__paragraph">
+                A great product is more than what’s in the box. It’s a promise of premium performance, world-class support, and everything you expect from a trusted brand. It’s just one of many reasons why you’ll shop with confidence on Bose.com.
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- contacts -->
+      <section class="page__section contact-us" id="contacts">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+
+          <div class="grid">
+            <div class="grid__item grid__item--1-2 grid__item--onIpad-1-3 grid__item--onDesktop-1-5">
+              <form action="/" method="get">
+                <input
```

Comment:
add correct link to form and check how to update form after user click button 'send'. Inputs in the form should become empty again.

Reply:
+